src/branch_view.js

```diff
@@ -55,9 +55,12 @@ function renderHtmlBlob(parent, html, hasApp) {
 
 	banner_utils.addClass(banner, 'branch-banner-is-active');
 
-	setTimeout(function() {
-			banner.style.top = '0';
-	}, 100);
+	if(parent === document.body) {
+		parent.insertBefore(banner, parent.firstChild);
+	}
+	else {
+		parent.appendChild(banner);
```

Comment:
It will not conflict because the node placement will be overwritten. It is still a mistake that needs to be corrected for the sake of clarity. Nice catch!